origami/models/rtu/base.py

```diff
@@ -2,45 +2,33 @@
 from datetime import datetime
 from typing import Any, Optional
 
-from pydantic import BaseModel, Field, root_validator
+from pydantic import BaseModel, Field, model_validator
+from typing_extensions import Annotated
 
 
 class BooleanReplyData(BaseModel):
     # Gate will reply to most RTU requests with an RTU reply that's just success=True/False
     success: bool
 
 
-class BaseRTURequest(BaseModel):
+class BaseRTU(BaseModel):
```

Comment:
Make a base class for both RTU requests and responses to hold the newly respelled channel_prefix validator. 

BaseRTURequest then becomes just a pass.